source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -4233,8 +4233,14 @@ KX_PYMETHODDEF_DOC(KX_GameObject, addDebugProperty,
 	char *name;
 	int visible = 1;
 
-	if (!PyArg_ParseTuple(args,"s|i:debugProperty", &name , &visible))
+	static const char *kwlist[] = {"name", "debug", nullptr};
+	if (!PyArg_ParseTupleAndKeywords(
```

Comment:
I don't know if this function request keyword, `name` must be provided and after that there's only one value.